src/main/drivers/pwm_output.c

```diff
@@ -469,37 +469,40 @@ void servoDevInit(const servoDevConfig_t *servoConfig)
 #ifdef USE_BEEPER
 void pwmWriteBeeper(bool onoffBeep)
 {
-        if (!beeperPwm.io)
-            return;
-        if (onoffBeep == true) {
-            *beeperPwm.channel.ccr = (PWM_TIMER_1MHZ / freqBeep) / 2;
-            beeperPwm.enabled = true;
-        } else {
-            *beeperPwm.channel.ccr = 0;
-            beeperPwm.enabled = false;
-        }
+    if (!beeperPwm.io)
```

Comment:
I know it's just reformatting. But can you add a block here too please?

Reply:
Argh!!!